spec/support/matchers/accessibility.rb

```diff
@@ -193,16 +193,16 @@ def decorative_svgs(page)
   end
 
   match do |page|
-    expect(decorative_svgs(page)).to all satisfy { |img| img[:role] == 'img' }
+    expect(decorative_svgs(page)).to all satisfy { |img| !img[:'aria-hidden'].nil? }
   end
 
   failure_message do |page|
-    img_tags = decorative_svgs(page).reject { |img| img[:role] == 'img' }.
+    img_tags = decorative_svgs(page).reject { |img| !img[:'aria-hidden'].nil? }.
```

Comment:
maybe positive check instead of double negative?
```suggestion
    img_tags = decorative_svgs(page).select { |img| img[:'aria-hidden'].nil? }.
```

Reply:
Yeah, good call 👍